cli/command/service/update.go

```diff
@@ -694,7 +694,7 @@ portLoop:
 			if v, ok := portSet[portConfigToString(&port)]; ok {
 				if v != port {
 					fmt.Println("v", v)
-					return fmt.Errorf("conflicting port mapping between %v:%v/%s and %v:%v/%s", port.PublishedPort, port.TargetPort, port.Protocol, v.PublishedPort, v.TargetPort, v.Protocol)
+					return fmt.Errorf("conflicting port mapping between %v:%v/%s/%s and %v:%v/%s/%s", port.PublishedPort, port.TargetPort, port.Protocol, port.PublishMode, v.PublishedPort, v.TargetPort, v.Protocol, v.PublishMode)
```

Comment:
Actually this error message is misleading. `portConfigToString` now includes `PublishedPort, TargetPort, Protocol, PublishMode`. So this message would show `conflicting port mapping between A and A`. @vdemeester Shall we remove `if v != port` part here?